priam/src/main/java/com/netflix/priam/identity/InstanceIdentity.java

```diff
@@ -300,4 +308,40 @@ public void updateToken() throws Exception {
         myInstance.setToken(tokenManager.sanitizeToken(nodetool.getTokens().get(0)));
         instanceRegistry.update(myInstance);
     }
+
+    // filter other DC's
+    private List<PriamInstance> filteredRemote(List<PriamInstance> priamInstances) {
+        List<PriamInstance> local = Lists.newArrayList();
+        for (PriamInstance priamInstance : priamInstances) {
+            if (priamInstance.getLocation().equals(location)) {
+                local.add(priamInstance);
+            }
```

Comment:
Yes, this is a Java 7 project and the C* deploy uses Java 7.  Rather than change too much at once I'm keeping with Java 7 compatibility.